src/main/java/seedu/address/model/deck/Card.java

```diff
@@ -58,6 +112,12 @@ public Performance getPerformance() {
     public int getTimesReviewed() {
         return timesReviewed;
     }
+    public int getConsecutiveCorrect() {
```

Comment:
This method is quite hard to understand, can you write some comments or documentation for this?